Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/PaintBrushManipulator.cpp

```diff
@@ -89,30 +45,39 @@ namespace AzToolsFramework
         const AZ::Transform& worldFromLocal, const AZ::EntityComponentIdPair& entityComponentIdPair)
     {
         m_ownerEntityComponentId = entityComponentIdPair;
-        m_config.m_ownerEntityComponentId = entityComponentIdPair;
-        PaintBrushRequestBus::Handler::BusConnect(entityComponentIdPair);
 
         SetSpace(worldFromLocal);
 
+        // Make sure the Paint Brush Settings window is open
+        AzToolsFramework::OpenViewPane(PaintBrush::s_paintBrushSettingsName);
+
+        // Get the radius from the global Paint Brush Settings.
+        float radius = 0.0f;
+        PaintBrushSettingsRequestBus::BroadcastResult(radius, &PaintBrushSettingsRequestBus::Events::GetRadius);
+
         // The PaintBrush manipulator uses a circle projected into world space to represent the brush.
         const AZ::Color manipulatorColor = AZ::Colors::Red;
         const float manipulatorWidth = 0.05f;
         SetView(
-            AzToolsFramework::CreateManipulatorViewProjectedCircle(*this, manipulatorColor, m_config.m_radius, manipulatorWidth));
-
-        // Notify listeners that the configuration changed. This is used to synchronize the internal manipulator configuration
-        // with the visible editable one stored on the Editor component.
-        PaintBrushNotificationBus::Event(
-            m_ownerEntityComponentId, &PaintBrushNotificationBus::Events::OnIntensityChanged, m_config.m_intensity);
-        PaintBrushNotificationBus::Event(
-            m_ownerEntityComponentId, &PaintBrushNotificationBus::Events::OnOpacityChanged, m_config.m_opacity);
-        PaintBrushNotificationBus::Event(
-            m_ownerEntityComponentId, &PaintBrushNotificationBus::Events::OnRadiusChanged, m_config.m_radius);
+            AzToolsFramework::CreateManipulatorViewProjectedCircle(*this, manipulatorColor, radius, manipulatorWidth));
```

Comment:
suggestion: This doesn't have to be part of this change, but something all other manipulators do is allow a custom view to be set and do not hard code it inside the manipulator. In future we might want to consider moving this logic out of `PaintBrushManipulator.cpp` and instead add it where we instantiate it

Reply:
I don't think it makes sense to allow configurable views in this case, because the logic of the PaintBrushManipulator is innately tied to knowledge of what the view is. If we passed in squares or something else, MovePaintBrush wouldn't work, AdjustRadius wouldn't make sense, etc.